spec2vec/model_building.py

```diff
@@ -0,0 +1,129 @@
+from typing import List
+from typing import Tuple
+from typing import Union
+import gensim
+from spec2vec.utils import ModelSaver
+from spec2vec.utils import TrainingProgressLogger
+
+
+def train_new_word2vec_model(documents: List, iterations: Union[List, int], filename: str = None, **kwargs):
+    """Train a new Word2Vec model (using gensim). Save to file if filename is given.
+
+    Parameters
+    ----------
+    documents:
+        List of documents, each document being a list of words (strings).
```

Comment:
I expected a list of documents where each document is a list of (word, count) tuples. 
In the case of spectrum's I expected the peak/loss and intensity tuple, but when take documents list from a test and do `list(documents[0])` I get 

```python
['peak@0.0',
 'peak@1.0',
 'peak@2.0',
 'peak@3.0',
 'peak@4.0',
 'peak@5.0',
 'peak@6.0',
 'peak@7.0',
 'peak@8.0',
 'peak@9.0']
```

How are counts or intensitiies being handled?

Reply:
Intensities are not used for training the word2vec model. They will only come into play when the document vectors are calculated. Multiple counts are rare except if a word occurs several times due to the rounding to 1,2 decimals. In that case the word will appear multiple times in the list.

Reply:
OK